core/templates/pages/topic-editor-page/editor-tab/topic-editor-stories-list.component.html

```diff
@@ -83,15 +83,24 @@
         </a>
       </td>
       <td>
-        <a (click)="topicRights.canEditTopic() && openStoryEditor(summary.getId())" class="story-list-summary text-center">
+        <a (click)="topicRights.canEditTopic() && openStoryEditor(summary.getId())" class="story-list-summary chapter-count text-center">
           <span> {{ summary.getPublishedChaptersCount() }} Published</span>
-          <span> {{ summary.getTotalChaptersCount() }} Total</span>
+          <span *ngIf="summary.getTotalChaptersCount()!=summary.getPublishedChaptersCount()"> {{ summary.getTotalChaptersCount() - summary.getPublishedChaptersCount() }} Awaiting Publication</span>
         </a>
       </td>
       <td>
         <a (click)="topicRights.canEditTopic() && openStoryEditor(summary.getId())" class="story-list-summary text-left">
-          <span *ngIf="summary.getUpcomingChaptersCount()==1"> {{ summary.getUpcomingChaptersCount() }} upcoming launch (next 14 days)</span>
-          <span *ngIf="summary.getUpcomingChaptersCount()>1"> {{ summary.getUpcomingChaptersCount() }} upcoming launches (next 14 days)</span>
+          <span *ngIf="summary.getUpcomingChaptersCount()==1">
+            {{ summary.getUpcomingChaptersCount() }} upcoming launch (next {{ summary.getUpcomingChaptersExpectedDays()[0] }} day)
+          </span>
+          <span *ngIf="summary.getUpcomingChaptersCount()>1">
+            {{ summary.getUpcomingChaptersCount() }} upcoming launches (next
+            <span *ngFor="let day of summary.getUpcomingChaptersExpectedDays(); index as id">
+              {{ day }}
+              <span *ngIf="id!=summary.getUpcomingChaptersExpectedDays().length-1">,</span>
```

Comment:
Add a space on either side of ">" and on either side of "!=".

Ditto elsewhere -- the lack of this contributes to your conditional expressions being hard to read.

Reply:
Done